tests/test_modeling_hubert.py

```diff
@@ -579,3 +656,102 @@ def test_inference_ctc_batched(self):
             "sweat covered brion's body trickling into the tight loin cloth that was the only garment he wore",
         ]
         self.assertListEqual(predicted_trans, EXPECTED_TRANSCRIPTIONS)
+
+    def test_inference_keyword_spotting(self):
```

Comment:
Amazing tests!

Reply:
Let's just add a comment to the tests with high `atol` to be changed later once fairseq merged PR: https://github.com/pytorch/fairseq/pull/3572